crates/cli_bin/tests/patterns_test.rs

```diff
@@ -180,6 +180,32 @@ fn fails_recursion() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn fails_to_find_non_existent_file() -> Result<()> {
+    let (_temp_dir, temp_fixture_path) = get_fixture("patterns_test", false)?;
+    let test_yaml_path = temp_fixture_path.join(".grit/grit.yaml");
+    // Update it to a config with a non-existent file
```

Comment:
Instead of inline updating please just make a new fixture for this.

Reply:
ok no worries, that's done: feb04e4